vue/components/ui/molecules/table/table.vue

```diff
@@ -54,6 +54,7 @@
                         </slot>
                     </tr>
                 </slot>
+

```

Comment:
remove extra line, please review the PR before assign

Reply:
This comes from `prettier`